documentation/modules/ROOT/pages/operations/debezium-platform.adoc

```diff
@@ -0,0 +1,446 @@
+
+|conductor.image
+|Image for the conductor
+|quay.io/debezium/platform-conductor:<release_tag>
+
+|conductor.offset.existingConfigMap
+|Name of the config map used to store conductor offsets. If empty it will be automatically created.
+|""
+
+|database.enabled
+|Enable the installation of PostgreSQL by the chart
+|false
+
+|database.name
+|Name of the database used by the platform
+|postgres
+
+|database.host
+|Host of the database used by the platform
+|postgres
+
+|database.auth.existingSecret
+|Name of the secret where `username` and `password` - of the database used by the platform - are stored . If empty a secret will be created using the `username` and `password` properties.
+
+When this is used, you don't need to set `database.auth.username` and `database.auth.password`.
+|""
+
+|database.auth.username
+|Username of the database used by the platform
+|user
+
+|database.auth.password
+|Password of the database used by the platform
+|password
+
+|offset.reusePlatformDatabase
+|Pipelines will use database to store offsets. By default, the database used by the platform is used.
+If you want to use a dedicated one set this property to false
+|true
+
+|offset.database.name
+|Name of the database used by the platform for storing the offsets
+|postgres
+
+|offset.database.host
+|Host of the database used by the platform for storing the offsets
+|postgres
+
+|offset.database.port
+|Port of the database used by the platform for storing the offsets
+|5432
+
+|offset.database.auth.existingSecret
+|Name of the secret where `username` and `password` - of the database used by the platform for storing the offsets - are stored. If not set `offset.database.auth.username` and `offset.database.auth.password` will be used.
+
+When this is used, you don't need to set `offset.database.auth.username` and `offset.database.auth.password`.
+|""
+
+|offset.database.auth.username
+|Username of the database used by the platform for storing the offsets
+|user
+
+|offset.database.auth.password
+|Password of the database used by the platform for storing the offsets
+|password
+
+|schemaHistory.reusePlatformDatabase
+|Pipelines will use database to store schema history. By default, the database used by the conductor service is used. If you want to use a dedicated one set this property to false
+|true
+
+|schemaHistory.database.name
+|Name of the database used by the platform for storing the schema history
+|postgres
+
+|schemaHistory.database.host
+|Host of the database used by the platform for storing the schema history
+|postgres
+
+|schemaHistory.database.port
+|Port of the database used by the platform for storing the schema history
+|5432
+
+|schemaHistory.database.auth.existingSecret
+|Name of the secret where `username` and `password` - of the database used by the platform for storing the schema history -are stored. If not set `schemaHistory.database.auth.username` and `schemaHistory.database.auth.password` will be used.
```

Comment:
```suggestion
|Name of the secret that stores the `username` and `password` that the platform uses to authenticate with the database that stores the schema history. 
If you do not specify value, instead of using a secret to store credentials, the platform uses the values of the schemaHistory.database.auth.username` and `schemaHistory.database.auth.password` properties to authenticate with the database.
```